src/super_gradients/common/plugins/deci_client.py

```diff
@@ -45,38 +52,37 @@ class DeciClient:
     def __init__(self):
         if not client_enabled:
             logger.error(
-                "deci-lab-client or deci-common are not installed. Model cannot be loaded from deci lab."
-                "Please install deci-lab-client>=2.55.0 and deci-common>=3.4.1"
+                "deci-platform-client or deci-common are not installed. Model cannot be loaded from deci lab."
+                "Please install deci-platform-client>=5.0.0 and deci-common>=12.0.0"
             )
             return
 
-        self.api_host = env_variables.DECI_API_HOST
-        self.lab_client = DeciPlatformClient(api_host=self.api_host)
-        self.lab_client.login(token=env_variables.DECI_PLATFORM_TOKEN)
+        self.lab_client = DeciPlatformClient()
 
-    def _get_file(self, model_name: str, file_name: str) -> Optional[str]:
+    def _get_file(self, model_name: str, file_name: "AutoNACFileName") -> Optional[str]:
         """Get a file from the DeciPlatform if it exists, otherwise returns None
         :param model_name:      Name of the model to download from, as saved in the platform.
         :param file_name:       Name of the file to download
         :return:            Path were the downloaded file was saved to. None if not found.
         """
         try:
-            response, status, response_headers = self.lab_client.get_autonac_model_file_link_with_http_info(
-                model_name=model_name, file_name=file_name, super_gradients_version=super_gradients.__version__
+            download_link = self.lab_client.get_autonac_model_file_link(
+                model_name=model_name,
+                file_name=file_name,
+                super_gradients_version=super_gradients.__version__,
             )
-            etag = response_headers.get("etag", "")
-            download_link = response.data
         except ApiException as e:
             if e.status == 401:
                 logger.error(
-                    "Unauthorized. wrong token or token was not defined. please login to deci-lab-client " "by calling DeciPlatformClient().login(<token>)"
+                    "Unauthorized. wrong credentials or credentials not defined. "
+                    "Please provide credentials via environment variables (DECI_CLIENT_ID, DECI_CLIENT_SECRET)"
                 )
             elif e.status == 400 and e.body is not None and "message" in e.body:
                 logger.error(f"Deci client: {json.loads(e.body)['message']}")
             else:
                 logger.debug(e.body)
             return None
-
+        etag = "123"
```

Comment:
@roikoren755 I hard coded `etag = "123"` to test, but once the new version is released we need to get etag from
`download_link, etag = self.lab_client.get_autonac_model_file_link` as discussed